app/javascript/components/types.ts

```diff
@@ -183,7 +183,7 @@ type ExerciseCore = {
   isExternal: boolean
 }
 
-export type ExerciseDifficulty = 'easy'
+export type ExerciseDifficulty = 'easy' | 'medium' | 'hard'
```

Comment:
Should this be `DifficultyCategory` to be consistent with the naming used in the Ruby model? I can do a rename factor if you agree.

```suggestion
export type ExerciseDifficulty = 'easy' | 'medium' | 'hard'
```

Reply:
Yeah - that'd be helpful 🙂  I think it should be `ExerciseDifficultyCategory`.